driver-core/src/test/functional/com/mongodb/internal/operation/AsyncCommandBatchCursorTest.java

```diff
@@ -86,7 +90,7 @@ void setUp() {
 
         connectionSource = mock(AsyncConnectionSource.class);
         operationContext = mock(OperationContext.class);
-        timeoutContext = mock(TimeoutContext.class);
+        timeoutContext = new TimeoutContext(TimeoutSettings.create(MongoClientSettings.builder().timeout(3, TimeUnit.SECONDS).build()));
```

Comment:
I had to get rid of unnecessary mocking to make the tests pass with the new code in cursors.